Library/Formula/fb-infer.rb

```diff
@@ -0,0 +1,53 @@
+class FbInfer < Formula
```

Comment:
Why did you decide to go with `fb-infer` as the formula name? The Website is <http://fbinfer.com/> and the repository name is just `infer`. Most existing formulae for Facebook's tools use just the repository name, so I'd suggest to use simply `infer`. (This would also make it match the name of the installed binary.)